docs/docs/usage/scaladoc/scaladocDocstrings.md

```diff
@@ -0,0 +1,188 @@
+Since many things are the same as in the old scaladoc, some parts are reused from this [article](https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html)
+
+
+Scaladoc extends Markdown with additional features, such as linking
+to API definitions. This can be used from within static documentation and blog
+posts to provide blend-in content.
+
+
+## Where to put scaladoc
+
+Scaladoc comments go before the items they pertain to in a special comment block that starts with a /** and ends with a */, like this:
+
+```scala
+/** Start the comment here
+  * and use the left star followed by a
+  * white space on every line.
+  *
+  * Even on empty paragraph-break lines.
+  *
+  * Note that the * on each line is aligned
+  * with the second * in /** so that the
+  * left margin is on the same column on the
+  * first line and on subsequent ones.
+  *
+  * The closing Scaladoc tag goes on its own,
+  * separate line. E.g.
+  *
+  * Calculate the square of the given number
+  *
+  * @param d the Double to square
+  * @return the result of squaring d
+  */
+ def square(d: Double): Double = d * d
+```
+
+In the example above, this Scaladoc comment is associated with the method square since it is right before it in the source code.
+
+Scaladoc comments can go before fields, methods, classes, traits, objects and even (especially) package objects. Scaladoc comments for package objects make a great place to put an overview of a specific package or API.
+
+For class primary constructors which in Scala coincide with the definition of the class itself, a @constructor tag is used to target a comment to be put on the primary constructor documentation rather than the class overview.
+
+## Tags
+
+Scaladoc uses `@<tagname>` tags to provide specific detail fields in the comments. These include:
+
+### Class specific tags
+
+- `@constructor` placed in the class comment will describe the primary constructor.
+
+### Method specific tags
+
+- `@return` detail the return value from a method (one per method).
+
+### Method, Constructor and/or Class tags
+
+- `@throws` what exceptions (if any) the method or constructor may throw.
+- `@param` detail a value parameter for a method or constructor, provide one per parameter to the method/constructor.
+- `@tparam` detail a type parameter for a method, constructor or class. Provide one per type parameter.
+
+### Usage tags
+
+- `@see` reference other sources of information like external document links or related entities in the documentation.
+- `@note` add a note for pre or post conditions, or any other notable restrictions or expectations.
+- `@example` for providing example code or related example documentation.
+- `@usecase` provide a simplified method definition for when the full method definition is too complex or noisy. An example is (in the collections API), providing documentation for methods that omit the implicit canBuildFrom.
```

Comment:
It is not used anymore by the collections API.

I am personally in favor of removing support for `@usecase`, by the way, since it often produces incorrect Scala signatures.

Reply:
It was leftover from Scala2 documentation. We actually don't support that tag in new Scaladoc